src/components/Search.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import getImages from "../requests/getImages";
+import "../styles/search.css";
+
+const Search = ({ setSearchResults }) => {
+  const [value, setValue] = useState();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setSearchResults(await getImages(value));
+  };
+
+  return (
+    <>
```

Comment:
You could get away with removing the fragment here as you have all your markup enclosed in the `form` component